docs/1.1/app_developers_guide/creating_sawtooth_network.md

```diff
@@ -119,7 +119,7 @@ For more information, see [Step 9: Stop the Sawtooth Environment](docker#stop-sa
 ## Step 1: Download the Docker Compose File
 
 Download the Docker Compose file for a multiple-node network,
-[sawtooth-default-poet.yaml](https://github.com/hyperledger/sawtooth-core/blob/main/docker/compose/sawtooth-default-poet.yaml).
+[sawtooth-default-poet.yaml](https://github.com/hyperledger/sawtooth-core/blob/1-2/docker/compose/sawtooth-default-poet.yaml).
```

Comment:
This one should be 1-1 instead of 1-2.

Reply:
What's the reason for that, @agunde406? So far I tried using the `chime` images, on the advice of @scealiontach, with some success, which I mentioned on the Contributor's Meeting call earlier today. Just trying to understand the reasoning behind the different versions.

Reply:
If you look at the file path, this is a part of the 1-1 docs (not 1-2) so it may have options or changes that are not supported for version 1-1. I am not sure if the 1-1 docker file works still though.

This doc repo holds documentation for multiple version of Sawtooth, and we need to be careful what links we use in each versions. Clearly both the 1-1 documentation and 1-2 documentation has bugs since they were both pointing to main. But all documentation under the docs/1.1 folder should point to the 1-1 branch, while 1.2 should point to 1-2.

Reply:
Sorry, @agunde406 , I made changes to 1-1 files by mistake there. Removing that change.